backend/leads/views.py

```
@@ -785,47 +1020,55 @@ def post(self, request, *args, **kwargs):
+
     permission_classes = (IsAuthenticated,)
 
-    
     def get_object(self, pk):
         try:
-            return Company.objects.get(
-                pk=pk
-            )
+            return Company.objects.get(pk=pk)
         except Company.DoesNotExist:
             raise Http404
```

Ruff (0.14.6)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)